substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/BundleSupport.java

```diff
@@ -110,6 +115,54 @@ final class BundleSupport {
     static final String BUNDLE_OPTION = "--bundle";
     static final String BUNDLE_FILE_EXTENSION = ".nib";
 
+    static final Path CONTAINER_GRAAL_VM_HOME = Path.of("/graalvm");
+    boolean useContainer;
+    private String containerTool;
+    private String bundleContainerTool;
+    private String containerToolVersion;
+    private String bundleContainerToolVersion;
+    private String containerImage;
+    private String bundleContainerImage;
+    private Path dockerfile;
+    private Path bundleDockerfile;
+    private static final List<String> SUPPORTED_CONTAINER_TOOLS = List.of("podman", "docker");
+    private static final String DEFAULT_DOCKERFILE = "ARG BASE_IMAGE=container-registry.oracle.com/os/oraclelinux:8-slim" + System.lineSeparator() +
```

Comment:
This is a prime candidate for being a resource file. Instead of having this enormous string, create a file in
```
substratevm/src/com.oracle.svm.driver/resources/container-default/Dockerfile
```
copy the string contents there and load the contents of that file into the static final `DEFAULT_DOCKERFILE` with
`com.oracle.svm.driver.NativeImage#getResource`.

Reply:
Addressed in commit https://github.com/oracle/graal/pull/6496/commits/7fc1854f3f288eb18b7bcb7ef16fd407654fa960